README.md

```diff
@@ -88,15 +88,17 @@ Once setup with a Go development environment and Docker, you can:
 
 2.  Start the OpenShift server
 
-        $ make run
+        $ sudo make run
 
 3.  In another terminal window, switch to the directory and start an app:
 
         $ cd $GOPATH/src/github.com/openshift/origin
         $ export OPENSHIFTCONFIG=`pwd`/openshift.local.config/master/admin.kubeconfig 
         $ _output/local/go/bin/osc create -f examples/hello-openshift/hello-pod.json
+        $ _output/local/go/bin/osadm policy add-role-to-user admin admin -n default
 
-In your browser, go to [http://localhost:6061](http://localhost:6061) and you should see 'Welcome to OpenShift'.
+
+In your browser, go to [https://localhost:8443](https://localhost:8443) and you should see 'Welcome to OpenShift'. Use 'admin' as the user and 'admin' as the password.
```

Comment:
This is explaining how to access the port exposed by the running pod. You don't need to log in to do that